assets/airports/kaus.json

```diff
@@ -0,0 +1,4024 @@
+                ["jbu", 2],
+                ["n/fastGA", 1],
+                ["skw", 3],
+                ["swa", 19],
+                ["ual/short", 2]
+           ]
+        },
+        {
+            "origin": "KAUS",
+            "destination": "",
+            "category": "departure",
+            "route": "KAUS17R.AEROZ1.SJT",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 1.33,
+            "airlines": [
+                ["aay", 1],
+                ["asa", 2],
+                ["bvn", 3],
+                ["n", 20],
+                ["n/fastGA", 1],
+                ["skw", 2],
+                ["swa", 3]
+           ]
+        },
+        {
+            "origin": "KAUS",
+            "destination": "",
+            "category": "departure",
+            "route": "KAUS17L.AUS5.AGJ",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 1.04,
+            "airlines": [
+                ["aca/short", 1],
+                ["asa", 3],
+                ["bvn", 1],
+                ["n", 20]
+           ]
+        },
+        {
+            "origin": "KAUS",
+            "destination": "",
+            "category": "departure",
+            "route": "KAUS17R.BNDIA2.BNDIA",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.167,
+            "airlines": [
+                ["amx/short", 1],
+                ["swa", 1],
+                ["ups", 1],
+                ["xa/fastGA", 1]
+           ]
+        },
+        {
+            "origin": "KAUS",
+            "destination": "",
+            "category": "departure",
+            "route": "KAUS17L.SAYBR2.SAYBR",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.46,
+            "airlines": [
+                ["n/fastGA", 1],
+                ["swa", 10]
+           ]
+        },
+        {
+            "origin": "KAUS",
+            "destination": "",
+            "category": "departure",
+            "route": "KAUS17L.CWK6.ACT",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.96,
+            "airlines": [
+                ["bvn", 3],
+                ["n", 20]
+           ]
+        },
+        {
+            "origin": "",
+            "destination": "KAUS",
+            "category": "arrival",
+            "route": "IAH.BITER8.KAUS17L",
+            "altitude": [18000, 36000],
+            "speed": 320,
+            "method": "random",
+            "rate": 0.29,
+            "airlines": [
+                ["aal/short", 6],
+                ["n/fastGA", 1]
+           ]
+        },
+        {
+            "origin": "",
+            "destination": "KAUS",
+            "category": "arrival",
+            "route": "WINDU.SEWZY4.KAUS17L",
+            "altitude": [18000, 36000],

```

Comment:
36000 is way too high... by the time an E145 is overflying the field, it will only be at 16,000ft! Reduce the range here.